page-archives.php

```diff
@@ -388,6 +398,7 @@ function insertSponsors($bcbname)
     1224 => 'bcb17', 1057 => 'bcb16', 931 => 'bcb15',
     785 => 'bcb14', 636 => 'bcb13', 479 => 'bcb12',
     399 => 'bcb11', 324 => 'bcb10', 220 => 'bcb9', 3 => 'bcb8');
```

Comment:
The list of cats here has 1:1 mapping with the list above and should be kept in parity. If you are automating the list above, may this should be automated too. Else this list will still need to be edited every time which beats the point of your nice initiative :)